### PR TITLE
Over ride NPM dependencies

### PR DESCRIPTION
**Pull request by japneetlambdatest**

### CODE REVIEW

File: commands/utils/archive.js [hunk 1]
```diff
@@ -71,6 +71,24 @@ function archive_project(lt_config) {
       { cwd: process.cwd(), ignore: ignore_files },
       { prefix: "project/" }
     );
+    //OverRide NPM Dependencies
+    if (lt_config["run_settings"]["npm_dependencies"]) {
+      console.log("Overriding NPM Dependencies");
+      let rawdata = fs.readFileSync("package.json");
+
+      let package = JSON.parse(rawdata);
```
Comment (akshatlambdatest) [inline commands/utils/archive.js:79]: Add a try-catch block while using JSON.parse

File: commands/utils/archive.js [hunk 2]
```diff
@@ -71,6 +71,24 @@ function archive_project(lt_config) {
...
+      package["dependencies"] = lt_config["run_settings"]["npm_dependencies"];
```
Comment (akshatlambdatest) [inline commands/utils/archive.js:80]: Use dot notation instead of array one :)

**Review by akshatlambdatest — CHANGES_REQUESTED** [review]

File: commands/utils/archive.js (hunk 1 above)
Comment (japneetlambdatest) [reply commands/utils/archive.js:79]: The json validation is already happening in a different file and code will not reach here is parsing fails, that's why error handling is not required here

Comment (akshatlambdatest) [reply commands/utils/archive.js:79]: It’s still a safe check to add :)

**Pull request merged**